Makefile

```diff
@@ -46,3 +46,14 @@ apiserver-generate: # generate api server resources, prod by default. ex: make a
 
 unifiedcatalog-generate: ## generate unified catalog resources
 	./scripts/unifiedcatalog/unifiedcatalog_generate.sh
+
+
+PROTOFILES := $(shell find $(WORKSPACE)/proto -type f -name '*.proto')
+PROTOTARGETS := $(PROTOFILES:.proto=.pb.go)
+
+%.pb.go : %.proto
+	@echo $<
+	@protoc  --proto_path=$(WORKSPACE)/proto --go-grpc_out=. --go_out=. $<
```

Comment:
I think we need to update this to properly output the files to the directory they are in now. I was doing it manually every time.

Reply:
Done